projects/examples/src/components/datagrid/datagrid-link.example.component.ts

```diff
@@ -0,0 +1,135 @@
+/*!
+ * Copyright 2019 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import { Component } from '@angular/core';
+import {
+    GridDataFetchResult,
+    GridColumn,
+    GridState,
+    ButtonConfig,
+    GridSelectionType,
+    ContextualButtonPosition,
+} from '@vcd/ui-components';
+import { InactiveButtonDisplayMode } from 'projects/components/src/datagrid';
```

Comment:
We should never directly import from a different project, always through  '@vcd/ui-components'. It often doesn't matter, but it creates two different classes and I've had problems before. We should add a tslint rule to prevent this...